Project.toml

```diff
@@ -40,7 +40,7 @@ RemoteFiles = "0.4"
 SPICE = "0.2"
 SatelliteToolbox = "0.9"
 TaylorIntegration = "0.8"
-TaylorSeries = "0.10"
+TaylorSeries = "0.10, 0.11"
```

Comment:
What about of extending this to include TaylorSeries v0.12, which is the most recent one?

Reply:
Currently, we cannot update `TaylorSeries` to the latest version v0.12 since `PlanetaryEphemeris` requires at most v0.11. Therefore, we need to update first in `PlanetaryEphemeris` and then in `NEOs`.

Reply:
TaylorSeries v0.12.2 includes a fix on `promotion_rule` that solves some ambiguities, and a bug in `tan`; the latter is more important in the context of PlanetaryEphemeris. I did a quick search for `tan` and I think it only appears in the computation of the osculating terms. If this is correct, there shouldn't be any problem to bump the version of TaylorSeries used in PlanetaryEphemeris and here, and we should perhaps have it 0.12 only.